src/app/app.module.ts

```diff
@@ -25,7 +28,8 @@ import { LoginPageComponent } from './components/login-page/login-page.component
     HttpClientModule
   ],
   providers: [
-    ApiService
+    ApiService,
+    GetSingleHeroService,
```

Comment:
This could just be called heroService, and we could put all hero related stuff in it?

Reply:
It's okay for me but i am not so sure that it should be like that. Tomorrow I will ask

Reply:
solved